### DISCUSSION TITLE
Start/end time of the experiment with download_datafiles comand

### DISCUSSION THREAD

**Discussion by fspinar** [NYUCCL/psiTurk#553] [Q&A] [answered]
Hi all,
I am trying to find the start/end time of each experiment the trialdata_csv or eventdata_csv files after running the download_datafiles command. I can see the timestamps of each trial, but I would like to get them in the date format (e.g., 11/2/2022, 1:42 PM) so that I can use the data in a specific time range.

How can I achieve this?

Thanks a lot!

**Comment by jacob-lee** [comment, marked as answer]
The timestamps are Unix time in milliseconds, using:

new Date().getTime()

Documentation for that function can be found here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime

To get Unix time in seconds, you'd divide by a thousand, and maybe round to
nearest integer, and feed that to some date-time class. Exactly what
depends on the language you are using to analyze the data. But you can
manually check using a variety of online tools (e.g.
https://www.epochconverter.com/ ; no affiliation).
